### PR TITLE
Leaderboard: hit the API

### PR DESCRIPTION
**Pull request by nelsinofrancisco**
# Implemented requirements

General requirements 

- [x] Make sure that there are no linter errors.
- [x] Make sure that you used the correct flow Gitflow.
- [x] Make sure that you documented your work in a professional way.

Project requirements

- [x] Create a new game with the name of your choice by using the API.
- [x] Implement the "Refresh" button (receiving data from the API and parsing the JSON).
- [x] Implement the form "Submit" button (sending data to the API).
- [x] Use async and await JavaScript features to consume the API.
- [x] No styling is required. 

#  ! IMPORTANT

> README.md file was updated only in this current branch, main and development branches don't have this change yet!

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,23 +7,25 @@
...
-        <input type="text" placeholder="Your name">
-        <input type="text" placeholder="Your score">
-        <button class="ms-auto" type="button">Submit</button>
+      <div class="right-content d-flex flex-column">
+        <div class="right-content-header">
+          <h2>Add your Score</h2>
+        </div>
+        <input id="input-user" type="text" placeholder="Your name">
+        <input id="input-score" type="text" placeholder="Your score">
+        <button id="add-score" class="ms-auto" type="button">Submit</button>
```
Comment (Emmyn5600) [inline src/index.html]: - [x] Score input type should be a number not a text.
arr32

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @nelsinofrancisco 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- POST and GET methods are working properly:heavy_check_mark:
- Data are being sent and received successfully:heavy_check_mark:
- Your project looks awesome :heavy_check_mark:

Changes required :writing_hand: 

       - Please Add a descriptive Readme file.
       - Please check the comment under the review.      

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by sinansevgi — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
------

Hi @nelsinofrancisco , 

Great work on making the changes requested by a previous reviewer 👏 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
- [x] You should always be sure your repository looks professional. Please  update your `readme` file **as requested by the previous TSE.** 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Happy Hacking! :computer: 

-----

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @nelsinofrancisco , 

Great work with readme file :clap:
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**